src/modules/governance/ProposalsList.tsx

```diff
@@ -79,8 +84,8 @@ export function ProposalsList({ proposals: initialProposals }: GovernancePagePro
         );
         setProposals((proposals) => {
           updatedProposals.map((proposal) => {
-            proposals[proposal.id].proposal = proposal;
-            proposals[proposal.id].prerendered = false;
+            proposals[proposals.length - 1 - proposal.id].proposal = proposal;
+            proposals[proposals.length - 1 - proposal.id].prerendered = false;
```

Comment:
Again, since the array is reversed this is the better solution that i found to access the correct proposal, since id and index are inverse now. Would love to use something like `proposal[-proposal.id]` but... yeah... js..